pkg/registry/authentication/tokenreview/storage.go

```diff
@@ -108,5 +108,11 @@ func (r *REST) Create(ctx context.Context, obj runtime.Object, createValidation
 		tokenReview.Status.Audiences = resp.Audiences
 	}
 
+	if createValidation != nil {
```

Comment:
do this on line 70, after we've validated but before we take action based on the review

Reply:
done